Waleed-abdullah/IoT-Fuzzing-FYP
# Connected with database

### Title: Connect Project to Amazon RDS PostgreSQL Database using Prisma
**Description:**
This pull request integrates the Amazon RDS PostgreSQL database with our project using Prisma as the ORM. This sets the foundation for our application to perform CRUD operations in a type-safe manner, thanks to Prisma and TypeScript.

### Changes:
**Prisma Setup:** Initialized Prisma and generated Prisma Client.
**Environment Variables:** Configured .env file to securely store Amazon RDS credentials.
**Database Migration:** Created initial migration files using prisma migrate dev.
**API Endpoints:** Added an API route (/api/getUsers) to fetch users from the database.
**Frontend:** Created a React component to display the list of users fetched from the database.

### How to Test:

- Pull the latest changes.
- Run npm install to install new dependencies.
- Set up your .env file with your Amazon RDS credentials.
- Start the development server with npm run dev.
- Navigate to http://localhost:3000 and you should see the list of users displayed.

### Checklist:
